sdk/cosmos/azure-cosmos-spark_3-1_2-12/src/main/scala/com/azure/cosmos/spark/BulkWriter.scala

```diff
@@ -212,11 +212,11 @@ class BulkWriter(container: CosmosAsyncContainer,
     var numberOfIntervalsWithIdenticalActiveOperationSnapshots = new AtomicLong(0)
     // Don't clone the activeOperations for the first iteration
     // to reduce perf impact before the Semaphore has been acquired
-    // this means if the semaphore can't be acquired within 1 minute
+    // this means if the semaphore can't be acquired within 10 minutes
     // the first attempt will always assume it wasn't stale - so effectively we
-    // allow staleness for one additional minute - which is perfectly fine
+    // allow staleness for ten additional minutes - which is perfectly fine
     var activeOperationsSnapshot = mutable.Set.empty[models.CosmosItemOperation]
-    while (!semaphore.tryAcquire(1, TimeUnit.MINUTES)) {
+    while (!semaphore.tryAcquire(10, TimeUnit.MINUTES)) {
```

Comment:
Is this part of another change?

Reply:
No - another issue that we discussed and decided to fix in this hotfix as well.